aQute.libg/src/aQute/lib/filter/Filter.java

```diff
@@ -14,6 +17,22 @@
 import java.util.Map;
 
 public class Filter {
+
+	// eliminate reflection on Version during resolver operations
+	static Class<?>			VERSION;
+	static MethodHandle	parseVersionMH;
+	static {
+		MethodHandles.Lookup publicLookup = MethodHandles.publicLookup();
+		try {
+			VERSION = Class.forName("org.osgi.framework.Version");
+			MethodType mt = MethodType.methodType(VERSION, String.class);
+			parseVersionMH = publicLookup.findStatic(VERSION, "parseVersion", mt);
```

Comment:
The prior version uses the Version constructor. You replace this with Version.parseVersion which has slightly different semantics. The Filter spec requires either the constructor or the static `valueOf` method. So you should use either `valueOf` (introduced in Framework 1.8 (R6)) or the constructor.

Reply:
Got it! will do.